INTEGRATION.md

```
@@ -0,0 +1,511 @@
+For ESLint, use a standard inline disable:
+
+```js
+os.abort(); // eslint-disable-line no-animal-violence/no-speciesist-language
+```
+
+**"canary deployment" in deployment tooling**
+
+The "canary" idiom is flagged at `warning` level in most tools. If your project's core subject matter is deployment patterns, disable the specific rule:
+
+In `.vale.ini`:
+```ini
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)